Bixal/move.mil
# Truncate command

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

Added the phone number paragraph to the location content type.
Added paragraph functionality to the command
Added dependency injection to the command.
Created a command to truncate all of the locations

## Testing

1. Pull changes
2. Install module
3. Run: 'docker-compose run php drupal parser' and check content page for location nodes
4. Run: 'docker-compose run php drupal truncate'
5. Run: 'docker-compose run php drupal database:client'
6. Run: "select * from node;" and check if there are any nodes of type location 

## Screenshots

_Attach relevant before and after screenshots here._
